lib/dolly/document.rb

```diff
@@ -132,5 +144,9 @@ def init_doc options
       self.doc['_id'] = self.class.namespace( normalized_id ) if normalized_id
     end
 
+    def valid_properties(options)
```

Comment:
this method should have a question mark at the end since it returns a boolean value